### ISSUE TITLE
dbo:startDate is not equivalent to schema:startDate

### ISSUE THREAD

**Issue by pheyvaer** [dbpedia/mappings-tracker#96]
`<http://dbpedia.org/ontology/endDate> <http://www.w3.org/2002/07/owl#equivalentProperty> <http://schema.org/endDate>`. is incorrect. Both properties mean the same; however, the domain of `schema:startDate` is broader then `dbo:startDate`, so they should not be considered equivalent.

**Comment by VladimirAlexiev** [comment]
Indeed, `dbo:startDate rdfs:range xsd:date` and examination `select * {?x dbo:startDate ?y}` shows that's true. So changed to `rdfs:subPropertyOf`

**Comment by VladimirAlexiev** [comment]
http://mappings.dbpedia.org/index.php/OntologyProperty:StartDate
